SmartEVSE-3/data/update2.html

```diff
@@ -85,7 +88,10 @@
                 <td>Community&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
               <td>bleeding edge&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
               <td><a href="https://github.com/"
-                     onclick="location.href=this.href + OWNER_COMM + '/' + REPO_COMM + '/releases';return false;"><script>document.write(OWNER_COMM)</script></a></td>
+                      onclick="var fullUrl = this.href + OWNER_COMM + '/' + REPO_COMM + '/releases';
+                                this.title = fullUrl;  // Update hover text
+                                location.href = fullUrl;
+                                return false;"><script>document.write(OWNER_COMM)</script></a></td>
```

Comment:
Please align this code according to the change you made above. Small thing but it helps the code looking good!